integreat_cms/cms/utils/pdf_utils.py

```diff
@@ -83,7 +85,15 @@ def generate_pdf(region, language_slug, pages):
             # In any other case, take the region name
             title = region.name
     language = Language.objects.get(slug=language_slug)
-    filename = f"{pdf_hash}/{capfirst(settings.BRANDING)} - {language.translated_name} - {title}.pdf"
+    # Make sure, that the length of the filename is valid. To prevent potential
+    # edge cases, shorten filenames to 3/4 of the allowed max length.
+    ext = ".pdf"
+    try:
+        max_len = ((os.statvfs(settings.PDF_ROOT).f_namemax // 4) * 3) - len(ext)
+    except (FileNotFoundError, ZeroDivisionError):
```

Comment:
Just out of curiosity: How can this throw a `ZeroDivisionError`? Or did it just happen and you're unsure why? :slightly_smiling_face: